src/components/ebay-breadcrumbs/index.marko

```diff
@@ -10,6 +10,7 @@ static var ignoredAttributes = [
 ];
 
 $ input.toJSON = noop;
+$ let anyHref = input && input.items && input.items.some(element => element.href != null);
```

Comment:
Use `var` here. 
There's an issue with `let/const` in older marko versions in the template.